### DISCUSSION TITLE
None of the YouTube playlists are downloaded completely

### DISCUSSION THREAD

**Discussion by Korb** [ErrorFlynn/ytdlp-interface#167] [Q&A]
Please look at [my logs](https://drive.google.com/drive/folders/1PHdxslooLivRrHcMlrA6SD6gdiUzzCSR). I see 443 errors, but I don't understand how to deal with them? In addition to files with the `mp4`, `webm` and `webp` extensions, many files with the `part` extension of various sizes are created in the folder.

The videos from these playlists play normally in the browser.

- ytdlp-interface 2.13.2
- yt-dlp 2024.07.25
- ffmpeg 2024.07.29
- Microsoft Windows 10.0.19045.4717
- VPN is not used.

Also, please consider the possibility of automatically deleting all temporary files after an unsuccessful download. That is, make it so that you do not have to manually search for all the part-files and delete them yourself.

**Comment by ErrorFlynn** [comment]
Sorry for replying so late, I didn't notice this until now. Unfortunately, I have no idea why you got those connection errors. I don't think they have anything to do with yt-dlp or any of the other software you listed. There seems to be some kind of networking issue between your system and Google's servers. I can't really speculate beyond that.

>Also, please consider the possibility of automatically deleting all temporary files after an unsuccessful download. That is, make it so that you do not have to manually search for all the part-files and delete them yourself.

I've considered automatically removing the temp files, but when? The program has no way of knowing whether you want to keep the temp files because you intend to try again at some point (and resume the download instead of restarting it), or whether you want them removed because you've given up on the download. The only thing I could realistically do is add a button that does the cleanup when you click on it. I'm not 100% sold on the idea right now, but I'll keep it in mind for the future.

**Comment by Korb** [comment]
I suppose that the reasons for the errors may be related to the blocking of YouTube by the authorities of my country. But this is not certain.
